### ISSUE TITLE
Need to change  Inventory Type from Market to Dealer

### ISSUE THREAD

**Issue by mrdigital0** [trclassic92/tr-blackmarket#5]
In the following lines you need to change the Market to Dealer

if it's Set to Market  the inventory script will not give the weapon 100% health thus making the weapon unusable unless repaired first

`
-- BlackMarket Shop Event
RegisterNetEvent("tr-blackmarket:PistolShop", function()
	TriggerServerEvent("inventory:server:OpenInventory", "shop", "Dealer", Config.PistolShop)
end)

RegisterNetEvent("tr-blackmarket:SubMachineGunsShop", function()
	TriggerServerEvent("inventory:server:OpenInventory", "shop", "Dealer", Config.SubMachineGunShop)
end)

RegisterNetEvent("tr-blackmarket:ShotGunsShop", function()
	TriggerServerEvent("inventory:server:OpenInventory", "shop", "Dealer", Config.ShotGunShop)
end)

RegisterNetEvent("tr-blackmarket:AssaultWeaponsShop", function()
	TriggerServerEvent("inventory:server:OpenInventory", "shop", "Dealer", Config.AssaultWeaponsShop)
end)

RegisterNetEvent("tr-blackmarket:MiscellanceousShop", function()
	TriggerServerEvent("inventory:server:OpenInventory", "shop", "Dealer", Config.MiscellanceousShop)
end)
`

**Comment by trclassic92** [comment]
Huh? 

Do you have a video that backs up this claim? 

>

**Comment by NWRPCommunity** [comment]
Yes i will advise two clips one your original script and with my editing to dealer

**Comment by mrdigital0** [comment]
So i dug into it a bit more

Looks like newest version of qb-inventory introduced Durability 
 
when setting the shop type to "Dealer" it uses the itemdata.info.quality = 100 from the qb-inventory predefined however if you want to use a custom shop type  example as yours  was market you'd need to do this 

items = {
        [1] = {
            name = "weapon_pistol",
            price = 3000,
            amount = 100,
            info = {
            quality = 100, 
             },
            type = "item",
            slot = 1,
        },
}

must add quality = 100  like i did otherwise the weapons are given at 0 quality and thus   you get this error in the screenshot 
![20220927221224_1](https://user-images.githubusercontent.com/13575547/192671970-0f510e3e-bd86-4d67-a769-db4259a76b0c.jpg)

**Comment by trclassic92** [comment]
Merged PR
